Releasing minio-py failed during `make tests`

```
Collecting mccabe<0.7,>=0.6
  Downloading mccabe-0.6.1-py2.py3-none-any.whl (8.6 kB)
Collecting lazy-object-proxy>=1.4.0
  Downloading lazy_object_proxy-1.6.0-cp38-cp38-manylinux1_x86_64.whl (58 kB)
Collecting wrapt<1.13,>=1.11
  Downloading wrapt-1.12.1.tar.gz (27 kB)
Building wheels for collected packages: wrapt
  Building wheel for wrapt (setup.py): started
  Building wheel for wrapt (setup.py): finished with status 'done'
  Created wheel for wrapt: filename=wrapt-1.12.1-cp38-cp38-linux_x86_64.whl size=78562 sha256=8b6c2484ab28490deab1fc9f70fa8513026acbae6c4a68fca858eafb3af56e4e
  Stored in directory: /q/.cache/pip/wheels/5f/fd/9e/b6cf5890494cb8ef0b5eaff72e5d55a70fb56316007d6dfe73
Successfully built wrapt
Installing collected packages: wrapt, lazy-object-proxy, toml, pycodestyle, mccabe, isort, astroid, urllib3, pylint, nose, mock, certifi, autopep8
Successfully installed astroid-2.6.2 autopep8-1.5.7 certifi-2021.5.30 isort-5.9.1 lazy-object-proxy-1.6.0 mccabe-0.6.1 mock-4.0.3 nose-1.3.7 pycodestyle-2.7.0 pylint-2.9.1 toml-0.10.2 urllib3-1.26.6 wrapt-1.12.1
WARNING: You are using pip version 21.1.2; however, version 21.1.3 is available.
You should consider upgrading via the '/usr/bin/python -m pip install --upgrade pip' command.
Running checks
************* Module tests.functional.tests
tests/functional/tests.py:127:4: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:574:8: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:656:8: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:1608:8: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:1658:4: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:2028:43: E1134: Non-mapping value kwargs is used in a mapping context (not-a-mapping)
make: *** [Makefile:11: check] Error 6
```

After seeing this and running locally `make tests` I can reproduce the same problem.

```
 make tests
Installing required dependencies
WARNING: Keyring is skipped due to an exception: Failed to unlock the collection!
Requirement already up-to-date: autopep8 in /home/harsha/.local/lib/python3.8/site-packages (1.5.7)
Requirement already up-to-date: certifi in /home/harsha/.local/lib/python3.8/site-packages (2021.5.30)
Requirement already up-to-date: mock in /home/harsha/.local/lib/python3.8/site-packages (4.0.3)
Requirement already up-to-date: nose in /home/harsha/.local/lib/python3.8/site-packages (1.3.7)
Requirement already up-to-date: pylint in /home/harsha/.local/lib/python3.8/site-packages (2.9.1)
Requirement already up-to-date: urllib3 in /home/harsha/.local/lib/python3.8/site-packages (1.26.6)
Requirement already satisfied, skipping upgrade: toml in /home/harsha/.local/lib/python3.8/site-packages (from autopep8) (0.10.1)
Requirement already satisfied, skipping upgrade: pycodestyle>=2.7.0 in /home/harsha/.local/lib/python3.8/site-packages (from autopep8) (2.7.0)
Requirement already satisfied, skipping upgrade: isort<6,>=4.2.5 in /home/harsha/.local/lib/python3.8/site-packages (from pylint) (5.6.4)
Requirement already satisfied, skipping upgrade: astroid<2.7,>=2.6.2 in /home/harsha/.local/lib/python3.8/site-packages (from pylint) (2.6.2)
Requirement already satisfied, skipping upgrade: mccabe<0.7,>=0.6 in /usr/lib/python3/dist-packages (from pylint) (0.6.1)
Requirement already satisfied, skipping upgrade: wrapt<1.13,>=1.11 in /usr/lib/python3/dist-packages (from astroid<2.7,>=2.6.2->pylint) (1.11.2)
Requirement already satisfied, skipping upgrade: lazy-object-proxy>=1.4.0 in /home/harsha/.local/lib/python3.8/site-packages (from astroid<2.7,>=2.6.2->pylint) (1.4.3)
WARNING: You are using pip version 20.2.4; however, version 21.1.3 is available.
You should consider upgrading via the '/usr/bin/python3 -m pip install --upgrade pip' command.
Running checks
************* Module tests.functional.tests
tests/functional/tests.py:127:4: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:574:8: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:656:8: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:1608:8: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:1658:4: W0621: Redefining name 'exc' from outer scope (line 2041) (redefined-outer-name)
tests/functional/tests.py:2028:43: E1134: Non-mapping value kwargs is used in a mapping context (not-a-mapping)
make: *** [Makefile:11: check] Error 6
```

```sh
python --version
Python 3.8.6
```

--- a/tests/functional/tests.py
+++ b/tests/functional/tests.py
@@ -2025,7 +2025,8 @@ def main():
         if arg_list:
             args = ()
             kwargs = arg_list
-            _call_test(test_name, *args, **kwargs)
+            _call_test(
+                test_name, *args, **kwargs)  # pylint: disable=not-a-mapping
 
     # Remove temporary files.
     if not is_mint_env:
@@ -2038,6 +2039,6 @@ if __name__ == "__main__":
         main()
     except TestFailed:
         sys.exit(1)
-    except Exception as exc:  # pylint: disable=broad-except
-        print(exc)
+    except Exception as excp:  # pylint: disable=broad-except
+        print(excp)
         sys.exit(-1)

